[2024-05-21T14:34:10.508+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:34:10.570+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v07.get_name scheduled__2022-11-20T00:00:00+00:00 [queued]>
[2024-05-21T14:34:10.582+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v07.get_name scheduled__2022-11-20T00:00:00+00:00 [queued]>
[2024-05-21T14:34:10.583+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:34:10.621+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-20 00:00:00+00:00
[2024-05-21T14:34:10.629+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5047) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:34:10.631+0000] {standard_task_runner.py:63} INFO - Started process 5066 to run task
[2024-05-21T14:34:10.631+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v07', 'get_name', 'scheduled__2022-11-20T00:00:00+00:00', '--job-id', '1241', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpn74txk5u']
[2024-05-21T14:34:10.633+0000] {standard_task_runner.py:91} INFO - Job 1241: Subtask get_name
[2024-05-21T14:34:10.697+0000] {task_command.py:426} INFO - Running <TaskInstance: our_dag_with_python_operator_v07.get_name scheduled__2022-11-20T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:34:10.814+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v07' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2022-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-11-20T00:00:00+00:00'
[2024-05-21T14:34:10.815+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:34:10.863+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:34:10.864+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:34:10.878+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v07, task_id=get_name, run_id=scheduled__2022-11-20T00:00:00+00:00, execution_date=20221120T000000, start_date=20240521T143410, end_date=20240521T143410
[2024-05-21T14:34:10.926+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:34:10.948+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-21T14:34:10.953+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
